must-gather/collection-scripts/gather

```diff
@@ -22,25 +23,27 @@ log "must-gather logs are located at: '${LOGFILE_PATH}'"
 mkdir ${BASE_COLLECTION_PATH}/cache-dir||:
 export KUBECACHEDIR=${BASE_COLLECTION_PATH}/cache-dir
 
-# cluster-scoped resources
-cluster_resources=(ns/openshift-operator-lifecycle-manager)
+# namespaces
+namespace_resources=(openshift-operator-lifecycle-manager)
```

Comment:
Yes, both `namespace_resources` and `cluster_resources` are used for different things.
I separated the "old" `cluster_resources` variable in two because it was confusing having not-namespaced resources in the same variable that the namespaces (and those with the "`ns/`" before, and removing that part for getting some info later [1]).

[1] https://github.com/openshift/cluster-logging-operator/pull/2901/files#diff-e5c2a69519b4a8fddd46d182316399558c9fd41434650a1196c7a9beaa14ada0L74